src/App.tsx

```diff
@@ -1,14 +1,94 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { FilterStatus } from './types/filter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectTodos, setSelectTodos] = useState<Todo | null>(null);
+  const [filter, setFilter] = useState(FilterStatus.All);
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+
+  const delay = 300;
+
+  const filteredTodos = useMemo(() => {
+    let filteredTodo = todos;
```

Comment:
```suggestion
    let filteredTodos = todos;
```

Reply:
Not fixed 
<img width="839" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/64750958/0eb28917-403d-418f-8746-b3acc2f7e306">